engine/lib/configuration.php

```diff
@@ -275,6 +275,33 @@ function _elgg_load_site_config() {
 }
 
 /**
+ * Set up CONFIG->cookies. (this is for unit testing)
+ *
+ * @see phpunit/bootstrap.php
+ *
+ * @param stdClass $CONFIG Elgg's config object
+ * @access private
+ */
+function _elgg_configure_cookies($CONFIG) {
+	// set cookie values for session and remember me
+	if (!isset($CONFIG->cookies)) {
```

Comment:
Not really, but something to keep in mind for the future. Think return values not side effects. Much more testable and typeable that way!